scoring/static/scoring/scss/input.scss

```diff
@@ -48,3 +48,10 @@ input {
     background-repeat: no-repeat;
     background-position: right 10px center;
 }
+
+// CEUK donate button
+input#ceuk-donate {
+    -moz-transition: none;
+    -webkit-transition: none;
+    transition: none;
```

Comment:
Rather than these `transition` lines, you probably want something like this:

```scss
#ceuk-donate {
background: none;
color: inherit;
border: none;
}
```

(Removes the background and border, and thanks to the ultra-high specificity of the ID selector, this trumps even the hover effects on the more generic `input:hover` … and also sets the `color` to inherit the default dark text colour of the parent so that, in the unlikely situation that the button image fails to load (it’s a third-party resource – could happen!), the fallback alt text will be a legible black on pale green, rather than white on pale green.)